lib/unfollower.rb

```diff
@@ -50,7 +53,6 @@ def self.followers(limit = 100)
         puts "#{f.screen_name}"
         puts "\tCreated #{f.created_at} :: #{f.description} :: #{f.website}"
         io.write "#{f.screen_name}\n"
-        sleep(1)
```

Comment:
I ran it without a limit at one point and rate limited. I think I had it in there as a precautionary method. Not as necessary now without rate limit issue. If it encounters the rate limit again, it simply bails to the `rescue` block (which also wasn't present at the time). So yeah, it can go away.